test/test_skip_poll.js

```diff
@@ -1,4 +1,5 @@
 var expect = require('chai').expect;
+var json1 = require('ot-json1');
```

Comment:
Can we add a test for `rich-text` (and any other types we want to check) please.